apps/src/blockly/addons/cdoWorkspaceSvg.js

```diff
@@ -52,8 +91,12 @@ export default class WorkspaceSvg extends GoogleBlockly.WorkspaceSvg {
     return Blockly.mainBlockSpace.getMetrics().toolboxWidth;
   }
 
-  // Use visibility:hidden not display:none to hide the trashcan so that it still takes up space, which is important
-  // for how the lid opening works.
+  /**
+   * Use visibility:hidden not display:none for the toolbox contents so that
```

Comment:
i'm not sure i understand this comment -- we're only ever setting `visibility:visible` on the toolbox contents in this method

Reply:
ah yeah I guess this comment is sort of for both `showTrashcan` and `hideTrashcan`. `showTrashcan` sets `visibility:hidden` on the toolbox contents and `hideTrashcan` sets it back to `visibility:visible`